common/history/characters/0_dvg_lotharingia.txt

```diff
@@ -0,0 +1,162 @@
+﻿CHARACTERS = {

```

Comment:
[15:38:12][character_templates_utils.cpp:29]: Invalid HQ region_pampas in  file: common/history/characters/0_dvg_lotharingia.txt line: 5
[15:38:12][jomini_effect.cpp:129]: PostValidate of effect 'create_character' returned false at  file: common/history/characters/0_dvg_lotharingia.txt line: 5
[15:38:12][character_templates_utils.cpp:29]: Invalid HQ region_amazon in  file: common/history/characters/0_dvg_lotharingia.txt line: 33
[15:38:12][jomini_effect.cpp:129]: PostValidate of effect 'create_character' returned false at  file: common/history/characters/0_dvg_lotharingia.txt line: 33
[15:38:12][character_templates_utils.cpp:29]: Invalid HQ region_amazon in  file: common/history/characters/0_dvg_lotharingia.txt line: 53
[15:38:12][jomini_effect.cpp:129]: PostValidate of effect 'create_character' returned false at  file: common/history/characters/0_dvg_lotharingia.txt line: 53
[15:38:12][character_templates_utils.cpp:29]: Invalid HQ region_amazon in  file: common/history/characters/0_dvg_lotharingia.txt line: 74
[15:38:12][jomini_effect.cpp:129]: PostValidate of effect 'create_character' returned false at  file: common/history/characters/0_dvg_lotharingia.txt line: 74
[15:38:12][character_templates_utils.cpp:29]: Invalid HQ region_amazon in  file: common/history/characters/0_dvg_lotharingia.txt line: 95
[15:38:12][jomini_effect.cpp:129]: PostValidate of effect 'create_character' returned false at  file: common/history/characters/0_dvg_lotharingia.txt line: 95
[15:38:12][character_templates_utils.cpp:29]: Invalid HQ region_amazon in  file: common/history/characters/0_dvg_lotharingia.txt line: 119
[15:38:12][jomini_effect.cpp:129]: PostValidate of effect 'create_character' returned false at  file: common/history/characters/0_dvg_lotharingia.txt line: 119

Reply:
Not sure what the issue here is? The characters seem to show up just fine in their respective region HQs just fine for me

Reply:
These region names do not exist.
See \common\strategic_regions\south_america_strategic_regions.txt for the region names to use.